### DISCUSSION TITLE
Front and backchannel logout URLs

### DISCUSSION THREAD

**Discussion by Anderen2** [gravitational/teleport#28938] [Q&A] [answered]
I've connected Teleport Enterprise with a external OIDC provider, which has worked great. However I'm having issues finding a logout URL for Teleport to ensure that the user is signed out when signing out from the identity provider. 

I found some references to "/teleport-logout" as a potential front-channel logout URL, however this only gets me a "404 page not found". 

Where can I find the front, or more preferably the backchannel logout URLs for Teleport?

**Comment by webvictim** [comment, marked as answer]
I don't believe there is (currently) any way to force a logout from the Teleport side when an OIDC session is logged out - they're managed separately. Teleport's behaviour is that when you sign in with an OIDC-based identity, it creates a matching user entry in Teleport with the same TTL as the certificate which Teleport has issued. When the certificate's TTL expires, the user is automatically deleted.
